java/src/jmri/jmrix/loconet/LnReporter.java

```diff
@@ -79,38 +78,70 @@ public void message(LocoNetMessage l) {
     }
 
     /**
-     * Handle transponding message
+     * Handle transponding message passed to us by the LnReporting Manager
+     * @param l - incoming loconetmessage
      */
     void transpondingReport(LocoNetMessage l) {
-        // check address
-        int addr = ((l.getElement(1) & 0x1F) * 128) + l.getElement(2) + 1;
-        if (addr != getNumber()) {
-            return;
-        }
+        int addr;
+        boolean enter;
+        int loco;
+        IdTag idTag;
+        if (l.getOpCode() == LnConstants.OPC_MULTI_SENSE) {
+            // Regular multisense enter exit
+            // check address
+            addr = ((l.getElement(1) & 0x1F) * 128) + l.getElement(2) + 1;
+            if (addr != getNumber()) {
+                return;
+            }
 
```

Comment:
If LnReporterManager does its job, this code should never be executed during normal operation.